engine/Dockerfile

```diff
@@ -18,13 +18,13 @@ RUN apk add bash \
 
 WORKDIR /etc/app
 COPY ./requirements.txt ./
-COPY ./grpcio-1.57.0-cp311-cp311-linux_aarch64.whl ./
+COPY ./grpcio-1.64.1-cp312-cp312-manylinux_2_17_aarch64.whl ./
```

Comment:
the `grpcio==1.57.0` package doesn't have any distributions built for Python 3.12 (pypi [files available for `1.57.0`](https://pypi.org/project/grpcio/1.57.0/#files)).. so therefore decided to bump this to latest version available.